### PR TITLE
Add chest

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/staged-recipes/issues/2795

Adds a package for the Python package [chest]( https://pypi.python.org/pypi/chest ).

### CODE REVIEW

File: recipes/chest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+about:
+  home: https://github.com/ContinuumIO/chest
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Simple on-disk dictionary'
+
+extra:
+  recipe-maintainers:
+    - mrocklin
```
Comment (jakirkham) [inline recipes/chest/meta.yaml]: Is it alright to add you as a maintainer here, @mrocklin?

**Comment by mrocklin** [comment]
Yes, though I don't plan to be active here.

**Comment by synapticarbors** [comment]
@jakirkham If you want to add me as an additional co-maintainer, I can keep and eye on this recipe. Otherwise, I'm happy to just merge it now.

**Comment by jakirkham** [comment]
Thanks @synapticarbors. Have added you.

**Pull request merged**